src/Console/PrintSchemaCommand.php

```diff
@@ -30,13 +29,11 @@ class PrintSchemaCommand extends Command
     /**
      * Execute the console command.
      *
-     * @param Repository $cache
-     * @param Filesystem $storage
-     *
-     * @throws DirectiveException
-     * @throws ParseException
+     * @param  \Illuminate\Cache\Repository  $cache
```

Comment:
I am a bit confused by this change in signature.

Is the Laravel code style really to not annotate Exceptions?

How about the FQCN, are they always done like that in Doc-Blocks?

Reply:
Laravel code styles include them.
Neverthless there is no `throw ...` in `\Nuwave\Lighthouse\Console\PrintSchemaCommand::handle`.

Other functions are responsible for throwing  DirectiveException or ParseException

Reply:
If F1 calls F2, that calls F3, that calls F4
and If F4 can throw an exception
F1, F2 & F3 are not annotated.

Reply:
PHPStorm hints at adding all Exceptions that occur somewhere down the function chain and are uncaught. I think that behaviour makes sense, as often the public methods of a class might not be the ones that actually throw the Exceptions, but it would be really inconvenient to have to dig in to the whole callstack to see what can happen.

Reply:
Up to you..
In this case there are really tons of functions that are missing these annotations

Reply:
I think we should be consistent first and foremost. Feel free to discuss in #531